apps/web/src/pages/templates/components/VariantsPage.tsx

```diff
@@ -85,10 +228,21 @@ export function VariantsPage() {
             variant={step}
             nodeType="variantRoot"
             data-test-id="variant-root-card"
+            errorMessage={
+              errorState?.variantIndex === variants.length
```

Comment:
Here is another place that would benefit from the `getRootVariantIndex` helper function